src/Http/Controllers/ApiController.php

```
@@ -175,25 +175,32 @@ public function getExperimentStats(Request $request, $experimentId)
     {
         try {
             $experiment = Experiment::findOrFail($experimentId);
-            
+            $deviceType = $this->resolveDeviceTypeFilter($request->query('device_type'));
+
             // Get variant statistics
             $variants = [];
             $controlRate = 0;
-            
+
             foreach ($experiment->variants as $variant => $weight) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$weight'. (undefined)

(UnusedLocalVariable)